docs/howto/manage-charms.md

```diff
@@ -209,23 +197,34 @@ juju deploy some-bundle --map-machines=existing,3=4,4=5
 
 ````
 
-Depending on the cloud substrate that your controller is running on, the above command will allocate a machine (physical,  virtual, LXD container) or a Kubernetes pod and then proceed to deploy the contents of the charm / bundle.
+````{dropdown} Tips for troubleshooting - machines
+Deploy on machines consists of the following steps: Provision resources/a machine M from the relevant cloud, via cloud-init maybe network config, download the `jujud` binaries from the controller, start `jujud`.
```

Comment:
This sounds odd to me. Deploy = application, and add = machine. However the steps do not include where the `jujud` agent downloads and installs the charm.   

Suggest either `adding a machine` to start, or adding `jujud agent downloads and installs the charm` to end?